src/makeRobot.js

```diff
@@ -38,7 +38,65 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  return {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return (`name: ${this.name}, `
+        + `chip version: ${this.version}, wheels: ${this.wheels}`);
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goRight(step = 1) {
+      if (step < 0) {
+        return this;
+      }
+      this.coords.x += step;
```

Comment:
nb: add empty line after statements
```suggestion

      this.coords.x += step;
```